channels/drdynvc/tsmf/ffmpeg/tsmf_ffmpeg.c

```diff
@@ -387,7 +406,7 @@ static tbool tsmf_ffmpeg_decode_audio(ITSMFDecoder* decoder, const uint8* data,
 #endif
 
 	if (mdecoder->decoded_size_max == 0)
-		mdecoder->decoded_size_max = AVCODEC_MAX_AUDIO_FRAME_SIZE + 16;
+		mdecoder->decoded_size_max = MAX_AUDIO_FRAME_SIZE + 16;
```

Comment:
`MAX_AUDIO_FRAME_SIZE` shouldnt be defined by us
we should find another way to backport this code